docs/contributing.rst

```diff
@@ -271,13 +271,20 @@ Here are the steps developers should take when doing a review.
 
 1. Assign yourself to the PR to let others know you're reviewing it.
 2. Familiarize yourself with the part of the codebase that the PR changes.
-3. Read through the code changes.
+3. Read through the code changes and commit messages.
 4. Test the PR branch.
-5. Make inline comments for minor changes.
-6. Make FIXUP commits for other changes.
+5. Make inline comments.
+6. Make commits for other changes.
```

Comment:
Maybe "proposed changes" instead of "other changes"? "other changes" just reads kind of ambiguously to me.